superagi/tools/duck_duck_go/requirements.txt

```diff
@@ -0,0 +1,4 @@
+pydantic==1.10.8
```

Comment:
this should go to main requirements. Why it is here? only duckduckgo-search is required in main.